src/_layouts/redirect.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <title>Redirecting&hellip;</title>
+  <link rel="canonical" href="{{ page.redirect.to }}">
+  <script>location = "{{ page.redirect.to }}" + location.hash</script>
+  <meta http-equiv="refresh" content="0; url={{ page.redirect.to }}">
+  <meta name="robots" content="noindex">
+  <h1>Redirecting&hellip;</h1>
+  <a href="{{ page.redirect.to }}">Click here if you are not redirected.</a>
```

Comment:
Guess it doesn't matter, but is it worth including the conventional `head` and `body` tags?

Reply:
I didn't even notice they weren't in there. I'll update to match the standards in the default.html